helm/ingress-azure/templates/deployment.yaml

```diff
@@ -30,6 +30,34 @@ spec:
       - name: {{ .Chart.Name }}
         image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
         imagePullPolicy: {{ .Values.image.pullPolicy }}
+        readinessProbe:
+          httpGet:
+            path: /health/ready
+        {{- if .Values.kubernetes }}
+        {{- if .Values.kubernetes.healthProbeServicePort }}
+            port: {{ .Values.kubernetes.healthProbeServicePort }}
+        {{ else }}
+            port: 8123
+        {{- end}}
+        {{ else }}
+            port: 8123
+        {{- end}}
+          initialDelaySeconds: 5
+          periodSeconds: 10
+        livenessProbe:
+          httpGet:
+            path: /health/alive
+        {{- if .Values.kubernetes }}
+        {{- if .Values.kubernetes.healthProbeServicePort }}
+            port: {{ .Values.kubernetes.healthProbeServicePort }}
+        {{ else }}
+            port: 8123
+        {{- end}}
+        {{ else }}
+            port: 8123
```

Comment:
same for this.

Reply:
Added a default value for `healthProbeServicePort`  in `values.yaml`